src/showcase/src/pages/loader.astro

```diff
@@ -0,0 +1,13 @@
+---
+import Screen from "../layouts/Screen.astro";
+---
+
+<Screen title="Loader" pageName="loader">
+  <script>
+    import { withLoader } from "$src/components/loader";
+    import { showMessage } from "$src/components/message";
+
+    // TODO: Fix loader image not loading
```

Comment:
leftover TODO?

Reply:
This is currently in `main`. I'll fix it in another PR.